webapp/sources/rudder/rudder-web/src/main/elm/rules/sources/ViewUtils.elm

```diff
@@ -538,3 +538,12 @@ generateLoadingList =
     ]
   , li[][i[][], span[][]]
   ]
+
+countRecentChanges: RuleId -> (Dict String (List Changes)) -> Float
+countRecentChanges rId changes =
+  case Dict.get rId.value changes of
+    Just cl ->
+      case List.Extra.last cl of
+        Just c -> c.changes
```

Comment:
Why `changes` is a `Float`?

Reply:
it should be a long (or equivalent). There's no way it holds a fraction